torch/distributed/elastic/rendezvous/utils.py

```diff
@@ -143,6 +145,21 @@ def _matches_machine_hostname(host: str) -> bool:
     return False
 
 
+def _delay(seconds: Union[float, Tuple[float, float]]) -> None:
+    """Suspends the current thread for ``seconds``.
+
+    Args:
+        seconds:
+            Either the delay, in seconds, or a tuple of a lower and an upper
+            bound within which a random delay will be picked.
+    """
+    if isinstance(seconds, tuple):
+        seconds = random.uniform(*seconds)
+    # Ignore delay requests that are less than 10 milliseconds.
+    if seconds >= 0.01:
```

Comment:
Any reason in particular to ignore delays for less than 10ms?

Reply:
The overhead of performing a syscall (suspending the thread + context switch) for such a short amount of time is just not worth in our particular use case.